ui/v2.5/public/manifest.json

```diff
@@ -2,9 +2,18 @@
   "short_name": "Stash",
   "name": "Stash: Porn Organizer",
   "icons": [
+    {
+      "src": "stash_icon.svg",
+      "sizes": "any"
+    },
+    {
+      "src": "favicon.png",
+      "sizes": "256x256",
+      "type": "image/png"
+    },
     {
       "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
```

Comment:
There are many different devices that use different sizes. I don't see a reason to remove this.

Reply:
the icon file doesnt actually contain those diffrent versions and as mentioned on [MDN](https://developer.mozilla.org/en-US/docs/Web/Manifest/Reference/icons#sizes)
> When multiple icons are available, browsers may select the most suitable icon for a particular display context. For raster formats like PNG, specifying the exact available sizes is recommended.

Reply:
Chomes manifest Dev Tools also rasies warnings about them when all the sizes are included 
![image](https://github.com/user-attachments/assets/91964c30-e5f5-42d9-9e08-4b8ce102e061)

Reply:
I've added them back anyway, despite chromes protests